### PR TITLE
Issue #6726: aligned javadoc/xdoc for SuppressWithNearbyCommentFilter

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6726 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/xdocs/config_filters.xml [hunk 1]
```diff
@@ -1799,18 +1799,19 @@ catch (Throwable th) { ... }
 &lt;/module&gt;
             </source>
             <source>
-public static final int lowerCaseConstant; // CHECKSTYLE IGNORE ConstantNameCheck FOR NEXT 3 LINES
```
Comment (pbludov) [inline src/xdocs/config_filters.xml]: The modifier `public` has been removed to fit the "100 character per line"  limit in .java counterpart.

File: src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java [hunk 2]
```diff
@@ -281,7 +456,7 @@ private void addTag(String text, int line) {
     /**
      * A Tag holds a suppression comment and its location.
      */
-    private static final class Tag {
+    /* package */ static class Tag {
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java]: Why is this changing?

File: src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java [hunk 3]
```diff
@@ -37,36 +37,207 @@
 
 /**
  * <p>
- * A filter that uses nearby comments to suppress audit events.
+ * Filter {@code SuppressWithNearbyCommentFilter} uses individual comments to suppress audit events.
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java]: I don't really agree with changing `nearby` to `individual`. `Nearby` is in the check name afterall.

File: src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java (hunk 2 above)
Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java]: Reverted.

File: src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java (hunk 3 above)
Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java]: Reverted. Usually when xml and javadoc comments collide, I take the xml part.
Typically, xml comments are more up-to-date.

Comment (rnveach) [reply src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java]: > Typically, xml comments are more up-to-date.

I agree, but I also look at what we are losing from the Javadoc side. If there is something of value there, then we should save it and not just throw it away.

File: src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java [hunk 4]
```diff
@@ -281,7 +456,7 @@ private void addTag(String text, int line) {
     /**
      * A Tag holds a suppression comment and its location.
      */
-    private static final class Tag {
+    private static class Tag {
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java]: Still a continuation of https://github.com/checkstyle/checkstyle/pull/6877#discussion_r300837966 ,
Why is this changing?

Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java]: Sorry, my bad. Now fixed.

**Pull request merged**